dll/win32/shell32/wine/pidl.c

```diff
@@ -1917,6 +1917,17 @@ BOOL _ILIsNetHood(LPCITEMIDLIST pidl)
     return FALSE;
 }
 
+BOOL _ILIsControlPanel(LPCITEMIDLIST pidl)
```

Comment:
ok since you've **added** the function into the wine-synced file, I've a question: Does this function exist in Wine? If so, is this added code its actual implementation (and not the deleted code from CDrivesFolder.cpp) ?

Reply:
@learn-more said it was ok as long as it was under the ```#ifdef __REACTOS__``` in an earlier comment. This is the same function I got from the CDrivesFolder.cpp

Reply:
Well, they differ a little, that's why I also asked.